src/notifications/endpoints/components/NewEndpointOverlay.tsx

```diff
@@ -14,10 +14,12 @@ import EndpointOverlayContents from 'src/notifications/endpoints/components/Endp
 // Constants
 import {NEW_ENDPOINT_DRAFT} from 'src/alerting/constants'
 import {NotificationEndpoint} from 'src/types'
+import ErrorBoundary from '../../../shared/components/ErrorBoundary'
 
 type ReduxProps = ConnectedProps<typeof connector>
 type Props = RouteComponentProps<{orgID: string}> & ReduxProps
 
+@ErrorHandling
 const NewRuleOverlay: FC<Props> = ({match, history, onCreateEndpoint}) => {
```

Comment:
A decorator wont work for function components.

Reply:
Got it.  Removed.